### PR TITLE
Rename AccessibleSVG to SVG and make it work with React Native

### PR DESCRIPTION
**Pull request by gziolo**
## Description
Follow-up: #9565.
Alternative to: #9294 and #9656.

## TODO

* [X] Make it work with the mobile app (React Native).

## How has this been tested?
`npm test`
`npm run build` - make sure all the SVG-based icons still work.

## Types of changes
Refactoring.

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation.

### CODE REVIEW

File: packages/components/src/dashicon/index.js [hunk 1]
```diff
@@ -900,7 +900,7 @@ export default class Dashicon extends Component {
 				height={ size }
 				viewBox="0 0 20 20"
 			>
-				<Path d={ path } />
+				<path d={ path } />
```
Comment (SergioEstevao) [inline packages/components/src/dashicon/index.js]: Shouldn't this continue to be Path? in order to get the React-Native version?

Comment (gziolo) [reply packages/components/src/dashicon/index.js]: I think temporary it should remain as is.

I was playing with `createElement` version which would do the replacement behind the scenes, but with no luck so far. I hope to do some more explorations soon.

**Comment by SergioEstevao** [comment]
But that will make our code in RN fail....

**Comment by gziolo** [comment]
@SergioEstevao I brought back Native components for SVG, can you double check if it works properly now? I hope to continue alternative solution where you can write DOM tags for SVG body but have RN to convert them to proper components behind the scenes. If it doesn't work we will have to ensure that all SVG use the pattern established so far.

File: packages/components/src/primitives/svg/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# SVG
+
+A drop-in replacement for the svg element that adds the required accessibility attributes for SVG elements across browsers.
+
+## Usage
+
+```jsx
+import { G, Path, SVG } from '@wordpress/components';
```
[the remark this answers is not in the document]
Comment (gziolo) [reply packages/components/src/primitives/svg/README.md]: I don't think it's essential to be include. I don't even know what it translates to :)

**Review by mcsf — APPROVED** [review]

**Comment by gziolo** [comment]
@mzorz, @hypest or @SergioEstevao, can you confirm that this PR works with https://github.com/wordpress-mobile/gutenberg-mobile/pull/129?

**Comment by hypest** [comment]
👋 @gziolo , I tested out https://github.com/wordpress-mobile/gutenberg-mobile/pull/129 against this branch and, apart from some updating that RN side branch needs, the app actually errors because the `paragraph` block has the `icon` being defined using plain, lowercased svg elements.

**Comment by gziolo** [comment]
Okey, let's merge this one and rebase #9294 against `master`.

**Comment by mtias** [comment]
Nice work following up here.

**Comment by mzorz** [comment]
> Okey, let's merge this one and rebase #9294 against master.

#9264 merged 🎉

**Pull request merged**